Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 70
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	7	aggregate_data_summary
	70	align
	1	all
	70	create_blocks
	70	merge_fastq
	218
Resources before job selection: {'_cores': 70, '_nodes': 9223372036854775807}
Ready jobs (77):
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
Selected jobs (70):
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	create_blocks
	aggregate_data_summary
	create_blocks
	create_blocks
	create_blocks
Resources after job selection: {'_cores': 0, '_nodes': 9223372036854775737}

[Tue Apr 22 12:34:12 2025]
rule create_blocks:
    output: results/Chip2/barcode26/4/fastq_files.lst
    jobid: 55
    wildcards: chip=Chip2, native=barcode26, block=4


        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode26/*.fastq.gz             | head -n 100 > results/Chip2/barcode26/4/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode26/4/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (4+1)))

        head -n $nhead results/Chip2/barcode26/4/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode26/4/fastq_files.lst
        rm -f results/Chip2/barcode26/4/fastq_files.lst.temp
        

[Tue Apr 22 12:34:12 2025]
rule create_blocks:
    output: results/Chip2/barcode27/2/fastq_files.lst
    jobid: 43
    wildcards: chip=Chip2, native=barcode27, block=2


        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/2/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/2/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (2+1)))

        head -n $nhead results/Chip2/barcode27/2/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/2/fastq_files.lst
        rm -f results/Chip2/barcode27/2/fastq_files.lst.temp
        

[Tue Apr 22 12:34:12 2025]
rule create_blocks:
    output: results/Chip1/barcode30/0/fastq_files.lst
    jobid: 1
    wildcards: chip=Chip1, native=barcode30, block=0


        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip1/*/fastq_pass/barcode30/*.fastq.gz             | head -n 100 > results/Chip1/barcode30/0/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip1/barcode30/0/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (0+1)))

        head -n $nhead results/Chip1/barcode30/0/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip1/barcode30/0/fastq_files.lst
        rm -f results/Chip1/barcode30/0/fastq_files.lst.temp
        

[Tue Apr 22 12:34:12 2025]
rule create_blocks:
    output: results/Chip1/barcode31/1/fastq_files.lst
    jobid: 12
    wildcards: chip=Chip1, native=barcode31, block=1


        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip1/*/fastq_pass/barcode31/*.fastq.gz             | head -n 100 > results/Chip1/barcode31/1/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip1/barcode31/1/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (1+1)))

        head -n $nhead results/Chip1/barcode31/1/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip1/barcode31/1/fastq_files.lst
        rm -f results/Chip1/barcode31/1/fastq_files.lst.temp
        

[Tue Apr 22 12:34:12 2025]
rule create_blocks:
    output: results/Chip2/barcode27/9/fastq_files.lst
    jobid: 50
    wildcards: chip=Chip2, native=barcode27, block=9


        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/9/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/9/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (9+1)))

        head -n $nhead results/Chip2/barcode27/9/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/9/fastq_files.lst
        rm -f results/Chip2/barcode27/9/fastq_files.lst.temp
        

[Tue Apr 22 12:34:12 2025]
rule create_blocks:
    output: results/Chip1/barcode31/6/fastq_files.lst
    jobid: 17
    wildcards: chip=Chip1, native=barcode31, block=6

Full Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 2071, in run_wrapper
    run(
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 125, in __rule_create_blocks
    input:
  File "/usr/lib/python3/dist-packages/snakemake/shell.py", line 162, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode26/*.fastq.gz             | head -n 100 > results/Chip2/barcode26/4/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode26/4/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (4+1)))

        head -n $nhead results/Chip2/barcode26/4/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode26/4/fastq_files.lst
        rm -f results/Chip2/barcode26/4/fastq_files.lst.temp' returned non-zero exit status 141.


        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip1/*/fastq_pass/barcode31/*.fastq.gz             | head -n 100 > results/Chip1/barcode31/6/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip1/barcode31/6/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (6+1)))

        head -n $nhead results/Chip1/barcode31/6/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip1/barcode31/6/fastq_files.lst
        rm -f results/Chip1/barcode31/6/fastq_files.lst.temp
        
Full Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 2071, in run_wrapper
    run(
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 125, in __rule_create_blocks
    input:
  File "/usr/lib/python3/dist-packages/snakemake/shell.py", line 162, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/2/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/2/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (2+1)))

        head -n $nhead results/Chip2/barcode27/2/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/2/fastq_files.lst
        rm -f results/Chip2/barcode27/2/fastq_files.lst.temp' returned non-zero exit status 141.

[Tue Apr 22 12:34:12 2025]
Error in rule create_blocks:
    jobid: 55
    output: results/Chip2/barcode26/4/fastq_files.lst
    shell:
        
        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode26/*.fastq.gz             | head -n 100 > results/Chip2/barcode26/4/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode26/4/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (4+1)))

        head -n $nhead results/Chip2/barcode26/4/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode26/4/fastq_files.lst
        rm -f results/Chip2/barcode26/4/fastq_files.lst.temp
        
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

[Tue Apr 22 12:34:12 2025]
Full Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 2071, in run_wrapper
    run(
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 125, in __rule_create_blocks
    input:
  File "/usr/lib/python3/dist-packages/snakemake/shell.py", line 162, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode26/*.fastq.gz             | head -n 100 > results/Chip2/barcode26/4/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode26/4/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (4+1)))

        head -n $nhead results/Chip2/barcode26/4/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode26/4/fastq_files.lst
        rm -f results/Chip2/barcode26/4/fastq_files.lst.temp' returned non-zero exit status 141.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 472, in _callback
    raise ex
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 458, in cached_or_run
    run_func(*args)
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 2101, in run_wrapper
    raise RuleException(
snakemake.exceptions.RuleException: CalledProcessError in line 96 of /data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile:
Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode26/*.fastq.gz             | head -n 100 > results/Chip2/barcode26/4/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode26/4/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (4+1)))

        head -n $nhead results/Chip2/barcode26/4/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode26/4/fastq_files.lst
        rm -f results/Chip2/barcode26/4/fastq_files.lst.temp' returned non-zero exit status 141.
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 96, in __rule_create_blocks


Error in rule create_blocks:
Full Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 2071, in run_wrapper
    run(
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 125, in __rule_create_blocks
    input:
  File "/usr/lib/python3/dist-packages/snakemake/shell.py", line 162, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/9/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/9/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (9+1)))

        head -n $nhead results/Chip2/barcode27/9/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/9/fastq_files.lst
        rm -f results/Chip2/barcode27/9/fastq_files.lst.temp' returned non-zero exit status 141.

[Tue Apr 22 12:34:12 2025]
RuleException:
CalledProcessError in line 96 of /data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile:
Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode26/*.fastq.gz             | head -n 100 > results/Chip2/barcode26/4/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode26/4/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (4+1)))

        head -n $nhead results/Chip2/barcode26/4/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode26/4/fastq_files.lst
        rm -f results/Chip2/barcode26/4/fastq_files.lst.temp' returned non-zero exit status 141.
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 96, in __rule_create_blocks
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    jobid: 43
rule create_blocks:
    output: results/Chip2/barcode28/9/fastq_files.lst
    jobid: 70
    wildcards: chip=Chip2, native=barcode28, block=9
[Tue Apr 22 12:34:12 2025]
    output: results/Chip2/barcode27/2/fastq_files.lst
Job failed, going on with independent jobs.

Error in rule create_blocks:
    shell:
        
        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/2/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/2/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (2+1)))

        head -n $nhead results/Chip2/barcode27/2/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/2/fastq_files.lst
        rm -f results/Chip2/barcode27/2/fastq_files.lst.temp
        
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode28/*.fastq.gz             | head -n 100 > results/Chip2/barcode28/9/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode28/9/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (9+1)))

        head -n $nhead results/Chip2/barcode28/9/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode28/9/fastq_files.lst
        rm -f results/Chip2/barcode28/9/fastq_files.lst.temp
        
    jobid: 50

    output: results/Chip2/barcode27/9/fastq_files.lst
Full Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 2071, in run_wrapper
    run(
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 125, in __rule_create_blocks
    input:
  File "/usr/lib/python3/dist-packages/snakemake/shell.py", line 162, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/2/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/2/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (2+1)))

        head -n $nhead results/Chip2/barcode27/2/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/2/fastq_files.lst
        rm -f results/Chip2/barcode27/2/fastq_files.lst.temp' returned non-zero exit status 141.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 472, in _callback
    raise ex
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 458, in cached_or_run
    run_func(*args)
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 2101, in run_wrapper
    raise RuleException(
snakemake.exceptions.RuleException: CalledProcessError in line 96 of /data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile:
Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/2/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/2/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (2+1)))

        head -n $nhead results/Chip2/barcode27/2/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/2/fastq_files.lst
        rm -f results/Chip2/barcode27/2/fastq_files.lst.temp' returned non-zero exit status 141.
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 96, in __rule_create_blocks

    shell:
        
        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/9/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/9/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (9+1)))

        head -n $nhead results/Chip2/barcode27/9/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/9/fastq_files.lst
        rm -f results/Chip2/barcode27/9/fastq_files.lst.temp
        
        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)

Full Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 2071, in run_wrapper
    run(
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 125, in __rule_create_blocks
    input:
  File "/usr/lib/python3/dist-packages/snakemake/shell.py", line 162, in __new__
    raise sp.CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/9/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/9/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (9+1)))

        head -n $nhead results/Chip2/barcode27/9/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/9/fastq_files.lst
        rm -f results/Chip2/barcode27/9/fastq_files.lst.temp' returned non-zero exit status 141.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 472, in _callback
    raise ex
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 458, in cached_or_run
    run_func(*args)
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 2101, in run_wrapper
    raise RuleException(
snakemake.exceptions.RuleException: CalledProcessError in line 96 of /data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile:
Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/9/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/9/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (9+1)))

        head -n $nhead results/Chip2/barcode27/9/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/9/fastq_files.lst
        rm -f results/Chip2/barcode27/9/fastq_files.lst.temp' returned non-zero exit status 141.
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 96, in __rule_create_blocks

RuleException:
CalledProcessError in line 96 of /data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile:
Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/2/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/2/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (2+1)))

        head -n $nhead results/Chip2/barcode27/2/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/2/fastq_files.lst
        rm -f results/Chip2/barcode27/2/fastq_files.lst.temp' returned non-zero exit status 141.
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 96, in __rule_create_blocks
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
RuleException:
CalledProcessError in line 96 of /data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile:
Command 'set -euo pipefail;  realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode27/*.fastq.gz             | head -n 100 > results/Chip2/barcode27/9/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode27/9/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (9+1)))

        head -n $nhead results/Chip2/barcode27/9/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode27/9/fastq_files.lst
        rm -f results/Chip2/barcode27/9/fastq_files.lst.temp' returned non-zero exit status 141.
  File "/data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile", line 96, in __rule_create_blocks
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 57, in run
Job failed, going on with independent jobs.

[Tue Apr 22 12:34:12 2025]
Job failed, going on with independent jobs.
rule create_blocks:
    output: results/Chip1/barcode25/4/fastq_files.lst
    jobid: 25
    wildcards: chip=Chip1, native=barcode25, block=4


        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip1/*/fastq_pass/barcode25/*.fastq.gz             | head -n 100 > results/Chip1/barcode25/4/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip1/barcode25/4/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (4+1)))

        head -n $nhead results/Chip1/barcode25/4/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip1/barcode25/4/fastq_files.lst
        rm -f results/Chip1/barcode25/4/fastq_files.lst.temp
        

[Tue Apr 22 12:34:12 2025]
rule create_blocks:
    output: results/Chip2/barcode28/2/fastq_files.lst
    jobid: 63
    wildcards: chip=Chip2, native=barcode28, block=2


        realpath /data/data-transfer_KCRB/2025-04-15_HT27-2025-04-11_Run1/Chip2/*/fastq_pass/barcode28/*.fastq.gz             | head -n 100 > results/Chip2/barcode28/2/fastq_files.lst.temp

        nfiles=$(wc -l < results/Chip2/barcode28/2/fastq_files.lst.temp)
        nfiles_per_block=$((nfiles / 10))
        nhead=$((nfiles_per_block * (2+1)))

        head -n $nhead results/Chip2/barcode28/2/fastq_files.lst.temp | tail -n $nfiles_per_block > results/Chip2/barcode28/2/fastq_files.lst
        rm -f results/Chip2/barcode28/2/fastq_files.lst.temp
        

[Tue Apr 22 12:34:12 2025]
rule aggregate_data_summary:
    output: AGGRE/Chip1/barcode30/non_sequential_summary.csv, AGGRE/Chip1/barcode30/sequential_summary.csv
    jobid: 211
    wildcards: chip=Chip1, native=barcode30

Full Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snakemake/jobs.py", line 371, in shellcmd
    self.format_wildcards(self.rule.shellcmd)
  File "/usr/lib/python3/dist-packages/snakemake/jobs.py", line 822, in format_wildcards
    return format(string, **_variables)
  File "/usr/lib/python3/dist-packages/snakemake/utils.py", line 405, in format
    return fmt.format(_pattern, *args, **variables)
  File "/usr/lib/python3.8/string.py", line 163, in format
    return self.vformat(format_string, args, kwargs)
  File "/usr/lib/python3.8/string.py", line 167, in vformat
    result, _ = self._vformat(format_string, args, kwargs, used_args, 2)
  File "/usr/lib/python3.8/string.py", line 207, in _vformat
    obj, arg_used = self.get_field(field_name, args, kwargs)
  File "/usr/lib/python3.8/string.py", line 278, in get_field
    obj = getattr(obj, i)
AttributeError: 'InputFiles' object has no attribute 'sequential_summary'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/snakemake/__init__.py", line 561, in snakemake
    success = workflow.execute(
  File "/usr/lib/python3/dist-packages/snakemake/workflow.py", line 850, in execute
    success = scheduler.schedule()
  File "/usr/lib/python3/dist-packages/snakemake/scheduler.py", line 374, in schedule
    self.run(job)
  File "/usr/lib/python3/dist-packages/snakemake/scheduler.py", line 389, in run
    self.get_executor(job).run(
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 356, in run
    super()._run(job)
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 174, in _run
    super()._run(job)
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 106, in _run
    self.printjob(job)
  File "/usr/lib/python3/dist-packages/snakemake/executors.py", line 112, in printjob
    job.log_info(skip_dynamic=True)
  File "/usr/lib/python3/dist-packages/snakemake/jobs.py", line 914, in log_info
    logger.shellcmd(self.shellcmd, indent=indent)
  File "/usr/lib/python3/dist-packages/snakemake/jobs.py", line 376, in shellcmd
    raise RuleException(str(ex), rule=self.rule)
snakemake.exceptions.RuleException: 'InputFiles' object has no attribute 'sequential_summary'

[Tue Apr 22 12:34:12 2025]
Finished job 70.
1 of 218 steps (0.46%) done
RuleException in line 163 of /data/Analysis/2025-04-15_HT27-2025-04-11_Run1/snakefile:
'InputFiles' object has no attribute 'sequential_summary'
unlocking
removing lock
removing lock
removed all locks
